spring-boot-admin-server-ui/src/main/frontend/services/instance.js

```diff
@@ -179,6 +179,19 @@ class Instance {
     return this.axios.get(uri`actuator/threaddump`);
   }
 
+  async downloadThreaddump(instanceName) {
+    return axios.get(uri`actuator/threaddump`, {
+      headers: {'Accept': 'text/plain'}
+    }).then( function (response) {
+      const fileSaver = require('file-saver');
+      const blob = new Blob([response.data], {type: 'text/plain;charset=utf-8'});
+      fileSaver.saveAs(blob, instanceName + '-threaddump.txt');
+    })
+      .catch(function (error) {
+        console.warn('Downloading threaddump failed: ', error);
```

Comment:
We should give some visible feedback if that fails, especially as there will be pre boot 2.2.0 applications which don't have support for this,

Reply:
I completely agree with you. Would you suggest propagating the error from the service to an actual component and then using some messaging mechanism to display it there?

Thanks!

Reply:
I'd propose not to catch the exception here but in the `threaddump/index.vue` and act on the error.

Reply:
Done.